src/asciidoctor-core-api.js

```diff
@@ -948,15 +958,45 @@ Title.$$proto.hasSubtitle = function () {
 
 // Inline API
 
-/** @namespace */
+/**
+ * @namespace
+ * @extends AbstractNode
+ */
 var Inline = Opal.Asciidoctor.Inline;
 
 /**
+ * Get the converted String text of this Inline node, if applicable.
+ *
  * @memberof Inline
- * @extends AbstractNode
+ * @returns {string} - returns the converted String text for this Inline node, or undefined if not applicable for this node.
  */
-Inline.$$proto.convert = function () {
```

Comment:
Is this intended ? Because we are using the method `convert` in the [smiley-inline-macro.js#L15](https://github.com/asciidoctor/asciidoctor.js/blob/c28dfa60de30bf43f4413bd208918fc976b6bdfd/spec/share/extensions/smiley-inline-macro.js#L15).